core/src/main/scala/fr/gospeak/core/services/storage/ProposalRepo.scala

```diff
@@ -56,13 +56,13 @@ trait SpeakerProposalRepo {
 }
 
 trait UserProposalRepo {
-  def list(user: User.Id, status: Proposal.Status, params: Page.Params): IO[Page[(Cfp, Proposal)]]
+  def listAcceptedProposals(user: User.Id, params: Page.Params): IO[Page[(Event, Proposal)]]
```

Comment:
why not keeping the previous one also ? (`def list(user: User.Id, status: Proposal.Status, params: Page.Params): IO[Page[(Cfp, Proposal)]]`)